### PR TITLE
Fix(DB/Spells): Add mana burn to Son of Flame

### LINKED ISSUE #9122: [Molten Core] Ragnaros Son of Flame not using Mana burn
### Current Behaviour

Son of flame does not use mana burn

### Expected Blizzlike Behaviour

Son of Flame should use mana burn to nearby targets.

### Source

https://classic.wowhead.com/guides/ragnaros-molten-core-strategy

### Steps to reproduce the problem

Go to MC
Ragnaros
Submerge phase
Son of Ragnaros should use mana burn.

### AC rev. hash/commit

009a52d1f13c

### Operating system

Win64

### Custom changes or Modules

Eluna

### PR DESCRIPTION
**Pull request by Nefertumm**
## Changes Proposed:
-  Add Lavashield (21857) aura to the npc.

## Issues Addressed:

- Closes #9122

## SOURCE:

From https://github.com/vmangos/core/blob/a01720f547f0d042b7989085a1a3558cc6e92a64/src/scripts/eastern_kingdoms/burning_steppes/molten_core/boss_ragnaros.cpp#L50

## Tests Performed:

- Ran the sql and tested in-game, the npc now drains mana to players nearby.

## How to Test the Changes:

Go to MC
Ragnaros
Submerge phase
Son of Ragnaros should use mana burn.

### CODE REVIEW

**Review by IntelligentQuantum — APPROVED** [review]

**Comment by Nyeriah** [comment]
Don't forget to update the labels please, without them things get lost

**Comment by Nyeriah** [comment]
@Nefertumm we need to check this after Andriuspel's PR

**Comment by Nefertumm** [comment]
> @Nefertumm we need to check this after Andriuspel's PR

Already checked, this one is not addressed in andriuspel's PR

File: data/sql/updates/pending_db_world/rev_1637176590610617700.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1637176590610617700');
+
+UPDATE `creature_template_addon` SET `auras` = '21857' WHERE `entry` = 12143;
```
Comment (Nyeriah) [inline data/sql/updates/pending_db_world/rev_1637176590610617700.sql:3]:
```suggestion
UPDATE `creature_template_addon` SET `auras` = 21857 WHERE `entry` = 12143;
```

**Comment by Nyeriah** [comment]
Tested, works

**Pull request merged**